src/views/Profile/AdminTable/AdminFilter.tsx

```diff
@@ -51,6 +50,30 @@ export const AdminFilter: React.FC<IMaterialsFilter> = ({
 
   const classes = useStyles();
   const { t } = useTranslation();
+  const [open, setOpen] = useState(false);
+  const selectRef = useRef(null);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      if (open) {
+        setOpen(false);
+      }
+    };
+
+    window.addEventListener('scroll', handleScroll);
+
+    return () => {
+      window.removeEventListener('scroll', handleScroll);
+    };
+  }, [open]);
+
+  const handleOpen = () => {
```

Comment:
if you wrap `handleOpen` and `handleClose` into `useCallback()` you could use `handleClose` in the `useEffect` above without risk to fall into infinite rerender